drake/addpath_drake.m

```diff
@@ -57,13 +57,19 @@
 addpath(fullfile(root,'solvers','BMI','util'));
 addpath(fullfile(root,'solvers','BMI','kinematics'));
 
-javaaddpath(fullfile(pods_get_base_path,'share','java','drake.jar'));
-
 if ispc 
   setenv('PATH',[getenv('PATH'),';',GetFullPath(pods_get_lib_path),';',fullfile(root,'pod-build','lib','Release'),';',fullfile(root,'pod-build','lib')]);
 end
 
 
+jarfiledir = fullfile(pods_get_base_path(), 'share', 'java');
```

Comment:
better add some comment here reminding us why you have to do this (possibly link back to the bug).